mobx/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: mobx
-version: 2.3.0+1
```

Comment:
The set_version command needs to be called from melos to also change the version.dart file for the package

Reply:
More details in this doc: https://mobx.netlify.app/development/release

Reply:
done

Reply:
formatting is in other pr, because there are several files being changed that are not related to current pr